pkg/daemon/daemon.go

```diff
@@ -46,6 +43,13 @@ func Main() {
 		os.Exit(1)
 	}
 
+	metricsAddr := os.Getenv("METRICS_BIND_ADDRESS")
```

Comment:
You might want this to be two separate variables - METRICS_HOST and METRICS_PORT.

Main reason being that users can then use the Kubernetes downward API to set the HOST portion to the node's IP address based on where the pod lands, without needing to bind to all addresses.